workspace/welcome-to-xod/015-xod-cloud/patch.xodp

```diff
@@ -0,0 +1,127 @@
+{
+  "comments": [
+    {
+      "content": "In this part of the tutorial, we'll only store the data in the cloud. In one of the next sections of the tutorial, we'll retrieve the data and react to it.\n\nYou can see a `cloud-store` node here. This node store data into your private feed. The feed is a storage of your data. The name of the feed is a path, like a path in URL or your filesystem. For example, `bob/smart-home-project/room1/temperature`.\n  This node prepends username and project name automatically to your feed. That's why a bound value on the `FEED` input is \"moisture\" instead of a full path.\n\nThe program is ready, but to make it works, you have to do a few things.\n\n## Excercise\n\n1. Log in into your XOD account in the Account Pane (main menu: View -> Toggle Account Pane).\n2. Open Project Preferences (main menu: Edit -> Project Preferences) ensure that the project has a name and click on the \"Generate\" button next to the \"XOD Cloud API Key\" input.\n3. Run a simulation and change the value of a `tweak-number` node a few times.\n4. Open the [feeds](https://xod.io/feeds/) page and check out that the data is stored in the feed named `YOUR_USERNAME/welcome-to-xod/moisture`.",
```

Comment:
In this part of the tutorial, we store the data in the cloud. In one of the next sections of the tutorial, we retrieve the data and react to it.

You can see a `cloud-store` node here. This node store data into your private feed. The feed is a storage of your data. The name of the feed is a path, like a path in URL or your filesystem. For example, `bob/smart-home-project/room1/temperature`.
This node prepends username and project name automatically to your feed. That's why a bound value on the `FEED` input is "moisture" instead of a full path.

The program is ready, but to make it works, you have to do a few things.

## Excercise

1. Log in into your XOD account in the Account Pane (main menu: View -> Toggle Account Pane).
2. Open Project Preferences (main menu: Edit -> Project Preferences). Ensure that the project has a name and click on the "Generate" button next to the "XOD Cloud API Key" input.
3. Run a simulation and change the value of a `tweak-number` node several times.
4. Open the [feeds](https://xod.io/feeds/) page and check out the data stored in the feed named `YOUR_USERNAME/welcome-to-xod/moisture`.